test/functional/crud_spec_tests.js

```diff
@@ -162,7 +206,7 @@ describe('CRUD spec', function() {
             .find({})
             .toArray()
             .then(collectionResults => {
-              expect(collectionResults).to.containSubset(outcome.result);
+              expect(collectionResults).to.containSubset(outcome.collection.data);
```

Comment:
I think we still need to check this in v1, see [this](https://github.com/mongodb/specifications/blob/master/source/crud/tests/v1/read/count.yml#L23) for example

Reply:
We are already asserting `outcome.results` [here](https://github.com/mongodb/node-mongodb-native/pull/2005/files/b9799403bdd02e81b51efc690b682c8f4663c281#diff-caa06aa6b4ca6f7d1cfcc135088b623fR198). This line is not asserting the result of the function call, but rather the state of the collection after the operation has been performed. If we were to use the example you provided above, we would be asserting that the entire collection would equal `3`.

Reply:
oh wow, yeah that's a bug. Do we need a more rigorous validator than `containsSubset`?

Reply:
containsSubset seems to default to `equals` for non-objects, so I think we're fine.